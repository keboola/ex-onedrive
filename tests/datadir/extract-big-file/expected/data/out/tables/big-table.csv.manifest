{"columns":["month","SA2_code","SA2_name","territorial_authority","total_dwelling_units","houses","apartments","retirement_village_units","townhouses_flats_units_other"]}
