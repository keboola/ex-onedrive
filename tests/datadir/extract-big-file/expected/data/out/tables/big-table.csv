"1990-04-01","100100","North Cape","Far North District","2","2","0","0","0"
"1990-04-01","100200","Rangaunu Harbour","Far North District","1","1","0","0","0"
"1990-04-01","100300","Inlets Far North District","Far North District","0","0","0","0","0"
"1990-04-01","100400","Karikari Peninsula","Far North District","2","2","0","0","0"
"1990-04-01","100500","Tangonge","Far North District","1","1","0","0","0"
"1990-04-01","100600","Ahipara","Far North District","2","2","0","0","0"
"1990-04-01","100700","Kaitaia East","Far North District","3","2","0","0","1"
"1990-04-01","100800","Kaitaia West","Far North District","0","0","0","0","0"
"1990-04-01","100900","Rangitihi","Far North District","0","0","0","0","0"
"1990-04-01","101000","Oruru-Parapara","Far North District","1","1","0","0","0"
"1990-04-01","101100","Taumarumaru","Far North District","2","2","0","0","0"
"1990-04-01","101200","Herekino-Takahue","Far North District","0","0","0","0","0"
"1990-04-01","101300","Peria","Far North District","0","0","0","0","0"
"1990-04-01","101400","Taemaro-Oruaiti","Far North District","0","0","0","0","0"
"1990-04-01","101500","Whakapaku","Far North District","0","0","0","0","0"
"1990-04-01","101600","Hokianga North","Far North District","0","0","0","0","0"
"1990-04-01","101700","Kohukohu-Broadwood","Far North District","0","0","0","0","0"
